scripts/postgres-debug.sh

```diff
@@ -0,0 +1,131 @@
+#    - Kubernetes Nodes, CPU, and Memory by cluster
+# 2. Edges Statistics:
+#    - Total Edge count
+#    - Edge count by cluster
+#    - Edges by edge type
+#    - Intercluster edge count by cluster (hub to cluster)
+#    - Total Inter-cluster edges count
+# 3. POSTGRESQL DEBUG DATA
+#    - POSTGRESQL configuration
+#    - POSTGRESQL database size
+#    - POSTGRESQL resources table size
+#    - POSTGRESQL edges table size
+#    - POSTGRESQL query activity
+#    - POSTGRESQL running queries
+#    - POSTGRESQL idle queries
+#    - POSTGRESQL index usage
+#    - POSTGRESQL vacuum and table stats
+
+psql -d search -U searchuser -c "SELECT NOW() as script_start_time;"
+psql -d search -U searchuser -c "SELECT version();"
+
+printf "\n\n----- COLLECTING RESOURCE STATS -----\n"
+
+printf "\n>>> Total resource count:\n\n"
+psql -d search -U searchuser -c "SELECT count(*) FROM search.resources;"
+
+printf "\n>>> Managed cluster count:\n\n"
+psql -d search -U searchuser -c "SELECT count(DISTINCT cluster) FROM search.resources;"
+
+printf "\n>>> Resource count by cluster:\n\n"
+psql -d search -U searchuser -c "SELECT cluster, count(uid) FROM search.resources GROUP BY cluster ORDER BY count DESC;"
+
+printf "\n>>> Resource count by apigroup and kind (top 25):\n\n"
+psql -d search -U searchuser -c "SELECT data->>'apigroup' as apigroup, data->>'kind' as kind, count(uid)
+    FROM search.resources GROUP BY apigroup,kind ORDER BY count DESC LIMIT 25;"
+
+printf "\n>>> Namespace count by managed cluster:\n\n"
+psql -d search -U searchuser -c "SELECT cluster, count(DISTINCT data->'namespace')
+    FROM search.resources GROUP BY cluster ORDER BY count DESC;"
+
+printf "\n>>> Resources by namespace (average, min, max):\n\n"
+psql -d search -U searchuser -c "SELECT avg(count) as avg, min(count) as min, max(count) as max
+    FROM (SELECT cluster, data->>'namespace' as namespace, count(uid)
+    FROM search.resources WHERE data->>'namespace' is not null GROUP BY cluster, namespace) as counts;"
+
+printf "\n>>> Kubernetes Nodes, CPU, and Memory by cluster:\n\n"
+psql -d search -U searchuser -c "SELECT cluster, data->'nodes' AS nodes, data->'cpu' AS cpu, data->>'memory' AS memory
+    FROM search.resources WHERE data->>'kind' = 'Cluster' ORDER BY nodes DESC;"
+
+
+printf "\n\n----- COLLECTING EDGES STATS -----\n"
+
+printf "\n>>> Total edges count:\n\n"
+psql -d search -U searchuser -c "SELECT count(*) FROM search.edges;"
+
+printf "\n>>> Edge count by cluster:\n\n"
+psql -d search -U searchuser -c "SELECT cluster, count(*) FROM search.edges GROUP BY cluster ORDER BY count DESC;"
+
+printf "\n>>> Edge count by type:\n\n"
+psql -d search -U searchuser -c "SELECT edgetype, count(*) FROM search.edges GROUP BY edgetype ORDER BY count DESC;"
+
+printf "\n>>> Intercluster edge count by cluster:\n\n"
+psql -d search -U searchuser -c "SELECT cluster, count(*) FROM search.edges WHERE edgetype = 'intercluster' GROUP BY cluster ORDER BY count DESC;"
```

Comment:
```suggestion
psql -d search -U searchuser -c "SELECT cluster, count(*) FROM search.edges WHERE edgetype = 'interCluster' GROUP BY cluster ORDER BY count DESC;"
```

Reply:
As per https://github.com/stolostron/search-v2-operator/blob/d71f894874c05740eee027a4dfaf6a89b71b0a62/controllers/create_pgconfigmap.go#L68. Have to change intercluster to interCluster